### ISSUE TITLE
PKC only shows 76 Movies which where unseen but there must be more

### ISSUE THREAD

**Issue by waxweazle2001** [croneter/PlexKodiConnect#1797]
Hey there,

I installed PKC because the ordinary Plex PlugIn don't want connect to my Plex Server anymore. 
I got a Windows Server with PMS installed and a Android Box with CoreElec.
I installed PKC and go to the Movie section and select "Plex Kodi Connect" and the Section "Last added" and Change the View to "Wall" I set the View to "Unseen movies" there are only 76 shown, if i go to the section "All Movies" there are 2434 in the List if I select "Unseen Movies" but i can't sort the list to last added Movies.
Are there any bugs known?

**Comment by croneter** [comment]
Use the normal Kodi library. Go to Movies. Select unwatched only. Happy?

**Comment by waxweazle2001** [comment]
No because in the normal Kodi Library as I Wrote I can't sort the list to "Last Added" I only can sort the list to "Date" but that's not the Same.

**Comment by croneter** [comment]
I guess you're using one of PKC's "nodes", such as `Add-ons` -> `Video add-ons` -> `PlexKodiConnect` -> `Name of your movie library` -> `Recently Added`. The number of items shown there is limited (!!) and depends on a PKC setting:
![image](https://user-images.githubusercontent.com/4637940/152837035-799829db-6aa1-4b99-82ea-18879a8af088.png)

If you use the normal, nativ Kodi movie library, simply sort by `Date added` to get the entire library sorted by the date the video was added. 
![image](https://user-images.githubusercontent.com/4637940/152837547-ed12e820-31dd-4d1e-864d-9bb44bd308aa.png)

**Comment by waxweazle2001** [comment]
Yes but I got no Nativ Library, I Used the Plex App before but it don't connect to my Server anymore and I don't know why.
Now I would use PKC, how must i bind in the Library in to get my Full Plex Librarys with my Categories etc.?
I Used the Aeon Nox: Silvio Skin, i searched for the Widget section but can't find the Point Movies in there. 
In my Kodi there ist no Native Video Library, when I bind the Movie Path into the Library, does PKC automatically set the Movie status to "seen" or "unseen" from the Plex Server and use the Covers from the Plex?
I got Kodi Matrix installed, is this an Problem with PKC?
I don't can read anything in the Wiki so I ask here.
